wp-content/themes/workingnyc/views/employer-programs/single.twig

```diff
@@ -55,15 +58,46 @@
     </div>
   </div>
   <div class="mb-7 mx-3 tablet-mx-14 desktop-mx-26">
-    <h3>Similar programs</h3>
-    <div class="grid grid-cols-2 gap-2 desktop:grid-cols-3 desktop:gap-x-2">
-      {% for program in post.meta('similar_programs') %}
-      <div>
-      {% include 'employer-programs/employer-program.twig' with { post: program } %}
-      </div>
-      {% endfor %}
-    </div>
+   
+    {#  slider  #}
+
+        <div class="nav flex mb-5">
+			    <h3>Similar programs</h3>
+            <a class="indicator ml-auto" href="#featureCarousel" role="button" data-bs-slide="prev">
+              <span class="fas fa-chevron-left" aria-hidden="true">
+                <svg aria-hidden="true" class="option__graphic m-auto border" tabindex="-1">
+                  <use href="#left-arrow"></use>
+                </svg>
+              </span>
+            </a>
+            <a class="w-aut indicator" href="#featureCarousel" role="button" data-bs-slide="next">
+              <span class="fas fa-chevron-right" aria-hidden="true">
+              <svg aria-hidden="true" class="option__graphic m-auto border" tabindex="-1">
+                <use href="#right-arrow"></use>
+              </svg>
+              </span>
+            </a>
+        </div>
+  <div class="container" id="featureContainer">
+		<div class="row my-auto justify-content-center">
+			<div id="featureCarousel" class="carousel slide" data-bs-ride="carousel">
```

Comment:
On tablet, only 2 cards should show, and on mobile, only 1 card should show

Reply:
I am using the width dynamically in jQuery so need to refresh the page in every screen size then you will get the correct number of cards. responsive css won't work here.

Reply:
Ok, that makes sense